source/views/news/noon-news-bulletin/row.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import * as React from 'react'
+import type {NewsBulletinType} from './types'
+import {ListRow, Title} from '../../components/list'
+import {Column, Row} from '../../components/layout'
+import {fastGetTrimmedText} from '../../../lib/html'
+import {AllHtmlEntities} from 'html-entities'
+
+type Props = {
+	bulletin: NewsBulletinType,
+}
+
+const entities = new AllHtmlEntities()
+
+export class NoonNewsRowView extends React.PureComponent<Props> {
+	render() {
+		const {bulletin} = this.props
+
+		return (
+			<ListRow arrowPosition="none">
+				<Row alignItems="center">
```

Comment:
Do we need this Row?

Reply:
Probably not. If we had more items, yes. I'll take it out.